### ISSUE TITLE
Security Audit?

### ISSUE THREAD

**Issue by padolph** [miloyip/rapidjson#636]
I plan on using rapidjson as part of a security protocol library. Has a source-level security audit of rapidjson been done? If not, are there any plans to do so?

**Comment by miloyip** [comment]
I do not know if anyone has done it before.
If you can help the community shall be appreciated.

**Comment by padolph** [comment]
Ok, thanks for the response. I will look into doing an audit as part of my project.

**Comment by w0o0o0oah** [comment]
@padolph did you ever complete the security audit?

**Comment by padolph** [comment]
@w0o0o0oah No, not directly. The larger project has had various audits, but none looked closely at rapidjson.
